composer.json

```diff
@@ -1,6 +1,7 @@
 {
     "name": "fusesource/stomp-php",
     "description": "stomp support for PHP",
+    "abandoned": true,
```

Comment:
I found this https://github.com/composer/composer/commit/320533bcf925228711e6533dfc0120248feea395 which made me feel it is